documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Technical change 1
+
+Description 1
+
+.Technical change 2
+
+Description 2
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.link:https://issues.redhat.com/browse/MTV-427[MTV-427] - OCP console plugin
+
+MTV Operator installed from CLI. +
+MTV web console URL.
+
+.link:https://issues.redhat.com/browse/MTV-426[MTV-426] - OpenStack migration
+
+See new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines.
+
+[id="known-issues-24_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.link:https://issues.redhat.com/browse/MTV-505[MTV-505] - Power off OpenStack VM in cold migration
+
+This issue implements the following: +
+
+` pkg/controller/plan/adapter/openstack/client.go` +
+` // Power off the source VM.` +
+` func (c *Client) PowerOff(vmRef ref.Ref) error` +
+` { return nil }`
+
+.link:https://issues.redhat.com/browse/MTV-349[MTV-349] - RHV snapshots are not deleted after a successful migration
+
+Issue description
+
+.link:https://issues.redhat.com/browse/MTV-456[MTV-456] - Migration fails during precopy/cutover while a snapshot operation is executed on the source VM
+
+Warm migration fails, returning an error, when performing another snapshot operation (create/delete) on the RHV source VM.
+Workaround: Wait for the snapshot operation to complete before running the next precopy/cutover procedure.
+
```

Comment:
I'd also keep the following known issues from 2.3:

- Deleting migration plan does not remove temporary resources.
- Unclear error status message for VM with no operating system
- Log archive file includes logs of a deleted migration plan or VM

I wouldn't keep `MTV 2.3.4 only: When the source provider is RHV, duplicating a migration plan fails in either the network mapping stage or the storage mapping stage.` - as it didn't happen with the new console (but I also don't think it is worth mentioning it as a resolved issue)

and I'd still keep `Migration of virtual machines with encrypted partitions fails during conversion` but with a different description:
The problem occurs for vSphere. Migrations from RHV and OpenStack don't fail, but the encryption key may be missing on the target OCP cluster.

Reply:
Known issues from 2.3 copied to beginning of 2.4